frontend/src/features/books/BookList.jsx

```diff
@@ -53,6 +53,7 @@ return (
               <td style={{ padding: '10px' }}>{book.author}</td>
               <td style={{ padding: '10px' }}>{book.description}</td>
```

Comment:
@nshelokar-vf, if CSS is being used in multiple places, please add common class and use accordingly.

Reply:
Okay sir.👍🏻